app/main.py

```diff
@@ -1,3 +1,54 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
```

Comment:
the `km` variable can be a `float` too.
So if you have python 3.10 then use `int | float` or Union[int, float] for earlier versions

Reply:
fixed